packages/design-tokens/design-token-previews/utils/constants.tsx

```diff
@@ -0,0 +1,111 @@
+import fontSizeTokens from "../../src/tokens/font-size.tokens.json";
```

Comment:
It isn't a utils file but the constant that defines the data for the entire table, I would keep it on the `TokenTable` level.